plans/multi-craft-labor-request-enhancement.md

```
@@ -0,0 +1,1924 @@
+      });
+    }
+  }
+
+  // Bulk insert notification records
+  await supabase
+    .from('labor_request_notifications')
+    .insert(notifications);
+}
+```
+
+**4.3 Consolidate Notifications for Multi-Craft Matches**
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)